src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age
+    && (<p className="Person__age">{`I am ${[person.age]}`}</p>)}
+    {person.isMarried && person.sex === 'm'
+    && (<p className="Person__partner">{`${person.partnerName} is my wife`}</p>)}
+    {person.isMarried && person.sex === 'f'
+    && (<p className="Person__partner">{`${person.partnerName} is my husband`}</p>)}
+    {!person.isMarried
+    && (<p className="Person__partner">I am not married</p>)}
```

Comment:
you can simplify this code

create `const partner = sex === 'm' ? 'wife' : 'husband';`

and write condition using this logic
```
   {isMarried
      ? `My ${partner}'s name is ${partnerName}`
      : 'I am not married'
    }

```